lib/components/key-handler.js

```diff
@@ -56,9 +57,7 @@ export default class KeyHandler extends React.Component {
   }
 
   handleKey(event: KeyboardEvent): void {
-    const keyCode = this.props.keyCode || keyNames(this.props.keyName);
-
-    if (event.keyCode !== keyCode) {
+    if (!checkKeyValues(event, this.props)) {
```

Comment:
We could probably name `checkKeyValues` a little better, what about `matchesKeyboardEvent`?